Makefile

```diff
@@ -308,7 +308,7 @@ tool-bootstrap: docker-build-bootstrap
 .PHONY: docker-build-bootstrap-transit
 docker-build-bootstrap-transit:
 	docker build -f cmd/Dockerfile --ssh default --build-arg TARGET=bootstrap/transit --build-arg COMMIT=$(COMMIT)  --build-arg VERSION=$(VERSION) --no-cache \
-	    --target production-transit-nocgo  \
```

Comment:
Should this change go to master as well?

Reply:
it is already in master..but i didn't find the separate commit to cherry pick